health/REFERENCE.md

```diff
@@ -482,8 +479,9 @@ Which in turn, results in the following behavior:
 ## Variables
 
 You can find all the variables that can be used for a given chart, using
-`http://your.netdata.ip:19999/api/v1/alarm_variables?chart=CHART_NAME` Example: [variables for the `system.cpu` chart of
-the registry](https://registry.my-netdata.io/api/v1/alarm_variables?chart=system.cpu).
+`http://NODE:19999/api/v1/alarm_variables?chart=CHART_NAME`, , replacing `NODE` with the IP address or domain name
```

Comment:
There are two commas after the URL, please, remove one.

Reply:
About the sentence `with the IP address or domain name`, I will give a suggestion, instead `domain name` use `hostname`.

Reply:
Fixed all these. Thanks for the suggestion.